Vocable/Common/Views/ToastView.swift

```diff
@@ -22,6 +24,12 @@ class PhraseSavedView: BorderedView {
             cornerRadius = frame.height / 2
         }
     }
+    
+    var text: String = "" {
+        didSet {
+            titleLabel.text = text
```

Comment:
```suggestion
            titleLabel?.text = text
```
Like the `awakeFromNib`, this ensures that we won't crash in the event that the xib gets lazily loaded and the text is set before titleLabel becomes available. It's not likely in the current implementation, but this class doesn't have knowledge of that :)

Reply:
Good call.